Polybar can't compile due to issue with upstream curl

[ 75%] Building CXX object bin/CMakeFiles/polybar.dir/utils/http.cpp.o
/home/gregf/.cache/pacaur/polybar-git/src/polybar/src/utils/http.cpp:2:10: fatal error: 'curl/curlbuild.h' file not found
#include <curl/curlbuild.h>
^~~~~~~~~~~~~~~~~~
[ 76%] Building CXX object bin/CMakeFiles/polybar.dir/utils/i3.cpp.o
1 error generated.
make[2]: *** [bin/CMakeFiles/polybar.dir/build.make:1359: bin/CMakeFiles/polybar.dir/utils/http.cpp.o] Error 1

Seems like <curl/buildcurl.h> has been removed from upstream and this is breaking the compilation. 
I tried removing the offending include statement and compilation was successful.

--- a/src/utils/http.cpp
+++ b/src/utils/http.cpp
@@ -1,5 +1,4 @@
 #include <curl/curl.h>
-#include <curl/curlbuild.h>
 #include <curl/easy.h>
 #include <sstream>
 

